src/components/app/details/cdDetails/service.ts

```diff
@@ -63,7 +63,7 @@ export interface DeploymentHistoryDetailRes extends ResponseType {
     result?: DeploymentHistoryDetail
 }
 
-const prepareDeploymentTemplateData = (rawData): Record<string, DeploymentHistorySingleValue> => {
+export const prepareDeploymentTemplateData = (rawData): Record<string, DeploymentHistorySingleValue> => {
     const deploymentTemplateData = {}
     if (rawData['templateVersion']) {
         deploymentTemplateData['templateVersion'] = { displayName: 'Chart Version', value: rawData['templateVersion'] }
```

Comment:
Can we extract  this `templateVersion` data from constant

Reply:
We're just trying to access the `templateVersion` value from rawData and it's happening only in this function. So I'm not changing anything here for now.